documentation/meta/maintenance/elasticsearch_cluster.md

```diff
@@ -349,55 +349,121 @@ best to pause it for performance reasons.
 
 ### Steps
 
-1. Use the Instance ID provided in the retirement notification to find the
-   "Private IPv4 address" of the instance. This value is the "Name" of the
-   ElasticSearch node.
-1. Identify where the node is stored in our Terraform state. This will be useful
+```{tip}
```

Comment:
Nice way to distinguish the names by `env` vs `environment`!

Reply:
Yeah, at one point I was getting errors because I entered 'prod' instead of 'production'. I guessed this kind of thing can be avoided with more documentation.